paper-toast.html

```diff
@@ -126,6 +124,36 @@
 
         properties: {
           /**
+           * The element to fit `this` into.
+           * Overridden from `Polymer.IronFitBehavior`.
+           */
+          fitInto: {
```

Comment:
This property is already defaulted to `window` in [`iron-fit-behavior`](https://github.com/PolymerElements/iron-fit-behavior/blob/master/iron-fit-behavior.html#L70), all I needed here is an observer.
If you see the previous override of `center`, it was using `fitInto` to **position** the toast.
Now, each time `fitInto` changes, I set `positionTarget` to equal `fitInto`.